ion/integration_test.go

```diff
@@ -0,0 +1,809 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package ion
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+// To debug/run one specific file in any of below paths, put the
+// file name with its extension. Works even if the file is listed in skip lists.
+// For example: const debugFile = "ints.ion"
+const debugFile = ""
```

Comment:
I thought it can be helpful for internal use when we are trying to fix the tests in the skip lists.
Removed.
And I will look into your suggestion for ```--debugFule=```

Reply:
Cool, thanks.